text/mavlink2_1.md

```diff
@@ -0,0 +1,62 @@
+  * Start date: 2022-10-12
+  * Contributors: Andrew Tridgell
+
+# Summary
+
+Rough proposal for MAVLink 2.1
+
+# Motivation
+
+Key features wanted:
+
+ * support for more than 256 system IDs
+ * support for checking CRCs in routers that don't have all xml
+ * better target_system/target_component support
+ * maximum compatibility with existing mavlink2 systems
+
+# Detailed Design
+
+The design takes advantage of the incompat_flags and compat_flags
+fields in the MAVLink2 header.
+
+New incompat_flags flags would be:
+
+ * MAVLINK_IFLAG_SYSID16 : system ID in header is 16 bits wide
```

Comment:
Another option is to use a variable length encoding for `sys_id`s greater than `255`.